cmd/traffic_layout/engine.h

```diff
@@ -44,11 +44,12 @@ struct RunrootEngine {
   int _argc;
   std::vector<std::string> _argv;
   // the flag for command line parsing
-  int help_flag    = 0;
-  int version_flag = 0;
-  int run_flag     = 0;
-  int clean_flag   = 0;
-  int force_flag   = 0;
+  int help_flag     = 0;
+  int version_flag  = 0;
+  int run_flag      = 0;
+  int clean_flag    = 0;
+  int force_flag    = 0;
+  int relocate_flag = 0;
```

Comment:
```C
bool foo_flag = false;
```

Reply:
These are bools. 
needs to be 
`bool help_flags = false;`